msg/sees_manual_control_data.msg

```diff
@@ -0,0 +1,7 @@
+uint64 timestamp                        	# time since system start (microseconds)
+
+uint8 sees_desired_control_source		# desired type of manual control source, RC (1) or Mav (2)
+uint64 valid_mavlink_setpoint_count		# number of Mavlink connections providing setpoints (implying joystick connected). Should be < 2.
+uint64 valid_rc_setpoint_count			# number of RC connections providing setpoints. Should be < 2.
```

Comment:
Similarly to `valid_mavlink_setpoint_count`, it might be good to change the type to `uint8_t` if the integer value is not large so the message size is not too large.

Reply:
Good spots, thanks!